<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuPrincipal</class>
 <widget class="QMainWindow" name="MenuPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(139, 69, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frmControles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	
border-radius:10px;
font:12pt&quot;Arial&quot;;
color:rgb(255,255,254);
border:2px solid #ffffff;
}
QPushButton:hover{
border-radius:10px;
font:12pt&quot;Arial&quot;;
color: rgb(255,255,255);
border:2px solid rgb(120,0,171);
  background-color: rgb(170, 85, 0);
}

#frmControles{
	background-image: url(:/fondo/imagen-llamativa-cintas-sobre-fondo-oscuro-generacion-ai_724548-22165.jpg);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnUsuarios">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 85, 0);</string>
     </property>
     <property name="text">
      <string>Usuarios</string>
     </property>
     <property name="icon">
      <iconset resource="recursos/images.qrc">
       <normaloff>:/prefijoNuevo/perfil-del-usuario.png</normaloff>:/prefijoNuevo/perfil-del-usuario.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEmpleados">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 85, 0);</string>
     </property>
     <property name="text">
      <string>Empleados</string>
     </property>
     <property name="icon">
      <iconset resource="recursos/images.qrc">
       <normaloff>:/prefijoNuevo/perfil-del-usuario.png</normaloff>:/prefijoNuevo/perfil-del-usuario.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDepartamentos">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 85, 0);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Departamentos</string>
     </property>
     <property name="icon">
      <iconset resource="recursos/images.qrc">
       <normaloff>:/prefijoNuevo/perfil-del-usuario.png</normaloff>:/prefijoNuevo/perfil-del-usuario.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnBienes">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 85, 0);
font: 12pt &quot;MS Shell Dlg 2&quot;;
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Bienes</string>
     </property>
     <property name="icon">
      <iconset resource="recursos/images.qrc">
       <normaloff>:/prefijoNuevo/perfil-del-usuario.png</normaloff>:/prefijoNuevo/perfil-del-usuario.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAsignacion">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 85, 0);</string>
     </property>
     <property name="text">
      <string>Asignacion Bienes</string>
     </property>
     <property name="icon">
      <iconset resource="recursos/images.qrc">
       <normaloff>:/prefijoNuevo/perfil-del-usuario.png</normaloff>:/prefijoNuevo/perfil-del-usuario.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAlmacenamiento">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>462</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 85, 0);
font: 14pt &quot;MS Shell Dlg 2&quot;;
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Almacenamiento</string>
     </property>
     <property name="icon">
      <iconset resource="recursos/images.qrc">
       <normaloff>:/prefijoNuevo/cerrar-con-llave.png</normaloff>:/prefijoNuevo/cerrar-con-llave.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnReportes">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>400</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 85, 0);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Reportes</string>
     </property>
     <property name="icon">
      <iconset resource="recursos/images.qrc">
       <normaloff>:/prefijoNuevo/cerrar-con-llave.png</normaloff>:/prefijoNuevo/cerrar-con-llave.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDesligar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>342</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 85, 0);
font: 11pt &quot;MS Shell Dlg 2&quot;;
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Desligar Bienes</string>
     </property>
     <property name="icon">
      <iconset resource="recursos/images.qrc">
       <normaloff>:/prefijoNuevo/perfil-del-usuario.png</normaloff>:/prefijoNuevo/perfil-del-usuario.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>621</width>
      <height>541</height>
     </rect>
    </property>
    <widget class="QWidget" name="subwindow">
     <property name="windowTitle">
      <string>Subwindow</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="mniUsuarios">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Usuarios</string>
   </property>
   <property name="toolTip">
    <string>Usuarios</string>
   </property>
  </action>
  <action name="mniEmpleados">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Empleados</string>
   </property>
   <property name="toolTip">
    <string>Emleados</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="recursos/images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnUsuarios</sender>
   <signal>clicked()</signal>
   <receiver>mdiArea</receiver>
   <slot>showFullScreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnEmpleados</sender>
   <signal>clicked()</signal>
   <receiver>mdiArea</receiver>
   <slot>showFullScreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
